templates/index.html

```diff
@@ -6,47 +6,66 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
+    <link rel="stylesheet" href="/static/style.css">
 </head>
 
 <body>
-<h1>Image Datestamp Tool</h1>
+    <div class="header">
+        <h1>Image Datestamp Tool</h1>
+        <a href="https://github.com/ch-iv/image-datestamp" target="_blank" class="github-icon" aria-label="View on GitHub">
+            <svg height="32" aria-hidden="true" viewBox="0 0 16 16" version="1.1" width="32" data-view-component="true">
+                <path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0016 8c0-4.42-3.58-8-8-8z"></path>
+            </svg>
+        </a>
+    </div>
 <h4>Adds datestamps to your images</h4>
-<form method=post enctype=multipart/form-data>
-    <label for="font">Font:</label>
-    <select name="font" id="font">
-        <option value="News Gothic Condensed Regular" selected="selected">News Gothic Condensed Regular</option>
-    </select>
+<form method="post" enctype="multipart/form-data">
+    <div class="form-group">
+        <div class="form-item">
+            <label for="font">Font:</label>
+            <select name="font" id="font">
+                <option value="News Gothic Condensed Regular" selected="selected">News Gothic Condensed Regular</option>
+                <option value="News Gothic Regular">News Gothic Regular</option>
+                <option value="News Gothic Bold">News Gothic Bold</option>
+                <option value="News Gothic Heavy">News Gothic Heavy</option>
```

Comment:
Currently, the backend doesn't support these font variations. Could you remove this for now? I will add this feature back when I get around to working on more fonts for this project.

Reply:
Got it, but could you handle this yourself? I'm not very free lately.